shiny/_main.py

```diff
@@ -172,11 +185,13 @@ def run_app(
     reload: bool = False,
     reload_dirs: Optional[list[str]] = None,
     reload_includes: list[str] | tuple[str, ...] = RELOAD_INCLUDES_DEFAULT,
+    reload_excludes: list[str] | tuple[str, ...] = RELOAD_EXCLUDES_DEFAULT,
     ws_max_size: int = 16777216,
     log_level: Optional[str] = None,
     app_dir: Optional[str] = ".",
     factory: bool = False,
     launch_browser: bool = False,
+    **kwargs: Dict[str, Any],
```

Comment:
This feels like we should make it a named argument of type `Dict[str, object]`. 

Ex:
```python
    unicorn_args: Dict[str, object]
```

I don't mind the name being explicit. By having a parameter, it can also give us opportunity for better typing down the road.

Reply:
If we are not changing the `kwargs` to a named arg, then the typing should be updated to represent the type of each element.

```suggestion
    **kwargs: object,
```

Reply:
The thought here is that it would let the user call something like this without wrapping the arguments in a dictionary. 

`shiny.run(app, uvicorn_arg1 = "foo", uvicorn_arg2 = "bar")` 

I'm happy with whatever, but since `shiny.run` is a pretty thin wrapper around `uvicorn.run` my intuition is that `*kwargs` is a bit better.

Reply:
Resolved on Slack, we're going with `*kwargs`